environment/src/test/java/com/sequenceiq/environment/environment/dto/AuthenticationDtoConverterTest.java

```diff
@@ -59,4 +67,19 @@ void authenticationToDto() {
                 .matches(m -> Objects.equals(m.getPublicKeyId(), environment.getPublicKeyId()))
                 .matches(m -> Objects.equals(m.isManagedKey(), environment.isManagedKey()));
     }
+
+    @Test
+    void testSshKeyCreation() {
+        String testKey = "ssh-rsa AAAASASFAS3532== banana@apple.com";
+        AuthenticationDto dto = AuthenticationDto.builder()
+                .withLoginUserName(LOGIN)
+                .withPublicKey(testKey)
+                .withPublicKeyId(PUBLIC_KEY_ID)
+                .withManagedKey(true)
+                .build();
+
+        EnvironmentAuthentication environmentAuthentication = underTest.dtoToAuthentication(dto);
+
+        assertEquals("ssh-rsa AAAASASFAS3532== cloudbreak", environmentAuthentication.getPublicKey());
```

Comment:
verify would be great

Reply:
can you help me, what do you mean by verify?